module/VuFindTheme/src/VuFindTheme/View/Helper/ThemeResources.php

```diff
@@ -161,22 +147,36 @@ protected function addLinks()
     }
 
     /**
-     * Add scripts to header.
+     * Add scripts to header or footer.
      *
-     * @return void
+     * @param string $position 'header' or 'footer'.
+     *
+     * @return string|null
      */
-    protected function addScripts()
+    protected function addScripts($position)
     {
         // Load Javascript (same ordering considerations as CSS, above):
-        $headScript = $this->getView()->plugin('headScript');
-        foreach (array_reverse($this->container->getJs()) as $current) {
-            $parts = $this->parseSetting($current);
-            $headScript()->forcePrependFile(
-                trim($parts[0]),
-                'text/javascript',
-                isset($parts[1])
-                ? ['conditional' => trim($parts[1])] : []
-            );
+        $js = array_reverse($this->container->getJs($position));
+        if ($position == 'header') {
+            $headScript = $this->getView()->plugin('headScript');
+            foreach ($js as $current) {
+                $headScript()->forcePrependFile(
+                    $current['file'],
+                    'text/javascript',
+                    $current['conditional'] ?? []
+                );
+            }
+        } elseif ($position == 'footer') {
+            $scripts = '';
+            foreach ($js as $current) {
+                // not sure if this call is correct, there seems to be lots of overhead
+                // no matter if setFile, appendFile, or prependFile is used
+                $scripts .= $this->getView()->plugin('inlineScript')->setFile($current['file'],
```

Comment:
I would add a line break before `$current['file']` here and indent the contents of the function call a bit less for readability.